lib/bot.js

```diff
@@ -120,6 +127,18 @@ const TwitchBot = class TwitchBot extends EventEmitter {
     this.irc.write(text + "\r\n")
   }
 
+
+
+  join (channel){
+    channel = parser.formatCHANNEL(channel);
+    this.writeIrcMessage(`JOIN ${channel}`);
+  }
+
+  part(channel){
+    channel = parser.formatCHANNEL(channel);
+    this.writeIrcMessage(`PART ${channel}`);
```

Comment:
should we remove the parted channel from `this.channels`?

Reply:
Probably not in this function call but with the confirmation response from the server. 
Can also be applied to the join-call and corresponding event.

Reply:
The same here as I mentioned with `join()`.